### PR TITLE
Demo Updates

### PR DESCRIPTION
**Pull request by bleonard**
* Makes data more predictable (no more randomness)
* Scale now handles decimals so you can run `roo demo --scale 0.1` to get a tenth of the users

### CODE REVIEW

File: plugins/@grouparoo/demo/src/util/sample_data.ts [hunk 1]
```diff
@@ -180,38 +184,47 @@ async function loadCsvTable(
     }
   };
 
-  for (let page = 1; page <= scale; page++) {
-    await fillTable(page);
+  const pages = Math.floor(scale);
+  const remainder = Math.round((scale - pages + Number.EPSILON) * 100) / 100;
```
Comment (evantahler) [inline plugins/@grouparoo/demo/src/util/sample_data.ts:188]: `Number.EPSILON` - that's a new one!

File: plugins/@grouparoo/demo/src/util/sample_data.ts [hunk 2]
```diff
@@ -299,24 +311,23 @@ function junkifyData(
...
-  } else {
-    v = Math.random() < 0.5 ? ` ${v} ` : "";
+  if (columns.length === 0) {
+    return;
   }
+  const keys = Object.keys(columns);
+  const key = keys[num % keys.length];
 
-  return v;
+  const empty = num % 3 === 0;
```
Comment (evantahler) [inline plugins/@grouparoo/demo/src/util/sample_data.ts:325]: Nice - switching between missing data and bad data somewhat deterministically

**Review by evantahler — APPROVED** [review]
This is getting to be a very fancy plugin.  Well done!

File: plugins/@grouparoo/demo/src/util/sample_data.ts (hunk 1 above)
Comment (bleonard) [reply plugins/@grouparoo/demo/src/util/sample_data.ts:188]: Stack OVerflow!

**Pull request merged**